browsermob-core/src/main/java/net/lightbody/bmp/BrowserMobProxy.java

```diff
@@ -537,4 +537,9 @@
      * @param filter filter instance
      */
     void addRequestFilter(RequestFilter filter);
+
+    /**
+     * Cleaning certificates on request
+     */
+    void cleanSslCertificat();
```

Comment:
Actually, I think this is specific to the Jetty implementation, and only needs to be in ProxyServer, not BrowserMobProxy. Then you can remove the method in BrowserMobProxyServer as well.